Classes/New Issue/NewIssueTableViewController.swift

```diff
@@ -205,6 +213,10 @@ final class NewIssueTableViewController: UITableViewController, UITextFieldDeleg
         bodyField.inputAccessoryView = actions
     }
 
+    func updateSubmitButtonState() {
```

Comment:
Would you like to write a test for this? :)

Reply:
We can do away with that function as I mentioned. But if we don't, I sure will!

Reply:
I'm concerned about the having split that onSend function in two and if I need to write a test for it. Also concerned about writing the test. I couldn't see anywhere where we are currently testing this portion of the code. I'm unfamiliar with tests in general so if you could give me some direction on this I would appreciate it.

Reply:
I'll get back to you on this one once I have a bit more time. Let's leave it for now; it's OK :)